README.md

```diff
@@ -7,57 +7,82 @@
 </p>
 
 <p align="center">
+    <a href="https://github.com/sulu/SuluRedirectBundle/blob/2.x/LICENSE" target="_blank">
+        <img src="https://img.shields.io/github/license/sulu/SuluRedirectBundle.svg" alt="GitHub license">
+    </a>
+    <a href="https://github.com/sulu/SuluRedirectBundle/releases" target="_blank">
+        <img src="https://img.shields.io/github/tag/sulu/SuluRedirectBundle.svg" alt="GitHub tag (latest SemVer)">
+    </a>
+    <a href="https://github.com/sulu/SuluRedirectBundle/actions" target="_blank">
+        <img src="https://img.shields.io/github/workflow/status/sulu/SuluRedirectBundle/Test%20application.svg?label=test-workflow" alt="Test workflow status">
+    </a>
+    <a href="https://github.com/sulu/sulu/releases" target="_blank">
+        <img src="https://img.shields.io/badge/sulu%20compatibility-%3E=2.0-52b6ca.svg" alt="Sulu compatibility">
+    </a>
+</p>
+<br/>
 
-<a href="https://github.com/sulu/SuluRedirectBundle/blob/2.x/LICENSE" target="_blank">
-    <img src="https://img.shields.io/github/license/sulu/SuluRedirectBundle.svg" alt="GitHub license">
-</a>
-<a href="https://github.com/sulu/SuluRedirectBundle/actions" target="_blank">
-    <img src="https://img.shields.io/github/workflow/status/sulu/SuluRedirectBundle/Test%20application.svg?label=test-workflow" alt="Test workflow status">
-</a>
-<a href="https://github.com/sulu/sulu/releases" target="_blank">
-    <img src="https://img.shields.io/badge/sulu%20compatibility-%3E=2.0-52b6ca.svg" alt="Sulu compatibility">
-</a>
+The SuluRedirectBundle adds simple but powerful capabilities for managing redirects to Sulu’s administration interface
+and allows content managers to manage redirects without any knowledge of web servers.
 
+<br/>
+<p align="center">
+    <img width="80%" src="https://sulu.io/uploads/media/800x@2x/02/342-SuluRedirectBundle%20Slideshow.gif?v=1-0" alt="SuluRedirectBundle Slideshow">
 </p>
+<br/>
+
+The SuluRedirectBundle is compatible with Sulu **starting from version 2.0**. Have a look at the `require` section in
+the [composer.json](https://github.com/sulu/SuluRedirectBundle/blob/2.x/composer.json) to find an
+**up-to-date list of the requirements** of the bundle.
+
+## 🚀&nbsp; Installation and Documentation
+
+Execute the following [composer](https://getcomposer.org/) commands to add the bundle to the dependencies of your
+project:
 
-The SuluRedirectBundle adds support for managing redirects in Sulu.
+```bash
+composer require sulu/redirect-bundle
+```
 
-Additional features included:
+Afterwards, visit the [bundle documentation](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc) to
```

Comment:
would be nice to add a `README.md` to the `Resources/doc` folder that contains a link for all sections 🙂

Reply:
```suggestion
Afterwards, visit the [bundle documentation](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc/README.md) to
```